Error while building on MacOS M1

**Describe the bug**
```
debug.cpp:1138:42: error: no member named '__srr0' in '__darwin_arm_thread_state64'
    return (void*) uc->uc_mcontext->__ss.__srr0;
                   ~~~~~~~~~~~~~~~~~~~~~ ^
```

**To reproduce**
Follow the steps to build KeyDB

**Expected behavior**
KeyDB builds successfully on MacOS M1

--- a/src/debug.cpp
+++ b/src/debug.cpp
@@ -1135,7 +1135,7 @@ static void *getMcontextEip(ucontext_t *uc) {
     #elif defined(__i386__)
     return (void*) uc->uc_mcontext->__ss.__eip;
     #else
-    return (void*) uc->uc_mcontext->__ss.__srr0;
+    return (void*) uc->uc_mcontext->__ss.__pc;
     #endif
 #elif defined(__APPLE__) && defined(MAC_OS_X_VERSION_10_6)
     /* OSX >= 10.6 */

